js/Common.js

```diff
@@ -1337,13 +1337,14 @@ CRM.strings = CRM.strings || {};
    * @return string
    */
   var currencyTemplate;
-  CRM.formatMoney = function(value, format) {
+  CRM.formatMoney = function(value, format, onlyNumber) {
```

Comment:
Coleman - why are you suggesting we swicth params?

CRM_Utils_Money::format() in 4.6 is: public static function format($amount, $currency = NULL, $format = NULL, $onlyNumber = FALSE, $valueFormat = NULL) {

So our suggested new parameter is in the same order.